bson/json_util.py

```diff
@@ -201,11 +209,13 @@ class JSONOptions(CodecOptions):
        <http://www.mongodb.org/display/DOCS/Mongo+Extended+JSON>`_.
 
     .. versionadded:: 3.4
+
```

Comment:
Add a ".. versionchanged:: 3.5" here to document when the canonical_extended_json option was added.